llvm/lib/Target/RISCV/RISCVInstrInfoXCheri.td

```diff
@@ -585,6 +585,10 @@ defm SC_C       : AMO_C_rr_aq_rl<"128", 0b00011, 0b100, "sc.c", GPR>;
 defm AMOSWAP_C  : AMO_C_rr_aq_rl<"128", 0b00001, 0b100, "amoswap.c", GPCR>;
 }
 
+let Predicates = [HasCheri], hasSideEffects = 0, mayLoad = 1, mayStore = 0 in
+def CLoadTags : RVInstCheriSrcDst<0x7f, 0x12, 0, OPC_CHERI, (outs GPR:$rd),
+                              (ins GPCR:$cs1), "cloadtags", "$rd, $cs1">;
```

Comment:
Belongs in a new "Tag-Memory Access Instructions" section just after "Adjusting to Compressed Capability Precision Instructions" (grouped based on the ISAv8 quick reference)